test/Quest.spec.ts

```diff
@@ -244,7 +244,7 @@
 //
 //             await expect(deployedQuestContract.claim(firstAddress.address, objectOfAddressesAndRewards[firstAddress.address], claim.proof)).to.be.revertedWithCustomError(
 //                 questContract,
-//                 'QuestPaused'
+//                 'Pausable: paused'
 //             )
 //         })
 //
```

Comment:
We should probably just remove this file. When we convert to an abstract contract we can add it back and test it directly then.